change/@fluentui-react-avatar-6355cdad-17ab-443b-8a0d-c8499ea34898.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 3 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27743&commitId=85f835a62ae0c8069d6ad7567d69484a60893629&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1930 screenshots unchanged, 0 screenshots with different dimensions and 3 screenshots with visible difference.


  <details><summary><b>unknown</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|AvatarGroup Converged.overflowContent - Dark Mode.popoverContentOpen.chromium.png|1825| Changed|
|AvatarGroup Converged.overflowContent - High Contrast.popoverContentOpen.chromium.png|2820| Changed|
|AvatarGroup Converged.overflowContent.popoverContentOpen.chromium.png|1985| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>

Reply:
changes are expected since it now uses border-box because of flex. It actually fixed a minor issue with the scrollbar :)